Api/Client.php

```diff
@@ -310,6 +307,72 @@ private function removeLastLineFromFiles($files = null, $noOfLinesToRemove = 1)
 
     /**
      *
+     * @param array $lines
+     * @param int   $noOfLinesToRemove
+     *
+     * @return $lines
+     */
+    private function removeLastLine(array $lines, $noOfLinesToRemove = 1)
+    {
+        $lastLine = count($lines) - $noOfLinesToRemove;
+        unset($lines[$lastLine]);
```

Comment:
This will only remove the specific line, while you want to remove all lines after this point